### PR TITLE
PR: Complete 5 Initial Tasks

### PR DESCRIPTION
**Pull request by 19neloyk**
Complete the 5 initial tasks outlined in the README

https://user-images.githubusercontent.com/25794113/211585580-770c80c9-c14c-4d3e-b252-b2b2272e7cb4.mov

### CODE REVIEW

File: pages/message/index.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+function MessagePage (){
+    // This deals with the state of the message that we simply
+    // show to the user, along with the relevant effect
+    const [regularMessage, setRegularMessage] = useState("");
+    useEffect( () => {
+        const doit = async () => {
+            let message = await getRegularMessage();
+            setRegularMessage(message);
+            console.log("HI")
```
Comment (jorgeochoareyes2) [inline pages/message/index.js:13]: Unnecessary console log.

File: pages/message/index.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+                </h2>
+            </main>
+    
+            
+        </div>
+        )
+};
+
+async function getRegularMessage() {
+    let response = await fetch('http://localhost:3000/api/message', {method: 'GET'});
```
Comment (jorgeochoareyes2) [inline pages/message/index.js:60]: Best to change the link to "/api/message" to prevent errors when we deploy to prod.

**Review by jorgeochoareyes2 — CHANGES_REQUESTED** [review]
Lets remove the console.log, replace the urls in fetch, and just one final thing we can have the message page be a file it self, no need to make it a nested route, this just helps make the pages folder less bloated.

**Comment by 19neloyk** [comment]
Just addressed your above comments.

**Review by jorgeochoareyes2 — APPROVED** [review]

**Pull request merged**